pallets/permissions/src/benchmarking.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
+
+use polymesh_primitives::{DispatchableName, PalletName};
+
+use frame_benchmarking::benchmarks;
+use frame_support::ensure;
+use sp_std::{iter, prelude::*};
+
+const MAX_PALLET_NAME: u32 = 512;
+const MAX_DISPATCHABLE_NAME: u32 = 1024;
+
+fn make_name(m: u32) -> Vec<u8> {
+    iter::repeat(b'x').take(m as usize).collect::<Vec<_>>()
+}
+
+benchmarks! {
+    _ {}
+
+    set_call_metadata {
+        let pallet_name: PalletName = make_name(MAX_PALLET_NAME).into();
```

Comment:
I think the length of the vector doesn't affect the storage read as it will take it as one storage read but still this function has the dependency on the length of the pallet name and dispatchable name so it should act as a catalyst variable, not the constant here.